src/Types.jl

```diff
@@ -483,6 +484,23 @@ getids(::Type{UnconstrainedParams}) = ids_free
 length(::Type{UnconstrainedParams}) =  6 + 2*Ia + 2*(B-1)*Ia + (D-1)*Ia + Ia-1
 
 
+# Parameters for a representation of the PSF
+type PsfParams <: Types.ParamSet
```

Comment:
I think this is the problem. This is already in the `Types` module, so this should be
```julia
type PsfParams <: ParamSet
```
no?